src/components/ExperimentCodeSetup.tsx

```diff
@@ -11,30 +34,150 @@ const useStyles = makeStyles((theme: Theme) =>
   }),
 )
 
-export default function ExperimentCodeSetup(): JSX.Element {
+export default function ExperimentCodeSetup({ experiment }: { experiment: ExperimentFull }): JSX.Element {
   const classes = useStyles()
+  const [tabValue, setTabValue] = useState(() => {
+    switch (experiment.platform) {
+      case Platform.Calypso:
+        return 0
+      default:
+        // most likely a landing page experiment
+        if (experiment.name.startsWith('lohp')) {
+          return 2
+        }
+        return 1
+    }
+  })
+
+  const defaultVariation = experiment.variations.flatMap((x) => (x.isDefault ? x.name : null))[0] ?? 'control'
+  const treatmentVariation = experiment.variations.flatMap((x) => (x.isDefault ? null : x.name))[0] ?? 'treatment'
```

Comment:
I suggest passing around variations as variation data structures rather than names as it makes this simpler and more extensible:

```suggestion
  const defaultVariation = experiment.variations.find(x => x.isDefault)
  const treatmentVariation = experiment.variations.find(x => !x.isDefault)
  
  // istanbul ignore next; typeguard, this is ensured by our schema
  if (!defaultVariation || !treatmentVariation) {
    throw new Error('Missing variations.')
  }
```


( 😅  It has taken me too long to figure out how to do these GitHub change suggestions.)